tools/js-sdk-release-tools/src/llc/generateRLCInPipeline/generateRLCInPipeline.ts

```diff
@@ -194,19 +194,6 @@ export async function generateRLCInPipeline(options: {
     const outputPackageInfo = getOutputPackageInfo(options.runningEnvironment, options.readmeMd, options.typespecProject);
 
     try {
-        // TODO: need to refactor
```

Comment:
Have we verified this?

Reply:
No, since the prod pipeline has no batch mode. Only ray can help run in piepline v2 that supports batch mode.
It's a save change for prod.